app/Check/Commands/SlackCheckStatusesCommand.php

```diff
@@ -73,6 +80,7 @@ protected function execute(
 		$options = [
 			'paused' => FALSE,
 			'this->project->maintenance' => NULL,
+			'this->project->notifications' => TRUE,
```

Comment:
Teď to nebude posílat notifikace do Slacku vůbec. Lepší by bylo, kdyby se tato podmínka teď přesunula jen na globální notifikaci do kanálu `#monitoring`, aby se posílaly notifikace konkrétním lidem.